### DISCUSSION TITLE
Networking Issues

### DISCUSSION THREAD

**Discussion by strongbad75** [ubenmackin/ACVM#17] [General]
I successfully built Windows 10 ARM VM usiing ACVM and installed the networking drivers per this guide (I see the Red Hat VirtIO NIC in device manager).  I have done this numerous times all with the same results as below.
Web browsing works, though Windows update or any other network traffic fail.
In the system tray, I have "white globe" in the system tray stating I have no internet access.

Your help is much appreciated.

**Comment by ubenmackin** [comment]
@strongbad75 This also happens with my installs. I think it is just an issue currently with how qemu does the networking, because we are not able to use bridging.

A few people over on the macrumors forums have the same issue (see https://forums.macrumors.com/threads/success-virtualize-windows-10-for-arm-on-m1-with-alexander-grafs-qemu-hypervisor-patch.2272354/post-29427668). Some of them state if you just let it sit, it will eventually come around. I haven't seen this happen yet on my install.

**Reply by strongbad75** [reply]
Thanks Ben for pointing this out.  We are still early and things will improve.

**Reply by joske** [reply]
I had this in my initial install. Reinstalled and then I never had it again. No idea why. Didn't do much differently I think.
